x-pack/legacy/plugins/lens/public/indexpattern_plugin/dimension_panel/dimension_panel.tsx

```diff
@@ -7,7 +7,9 @@
 import _ from 'lodash';
 import React from 'react';
 import { EuiFlexItem, EuiFlexGroup, EuiButtonIcon } from '@elastic/eui';
+import { Storage } from 'ui/storage';
 import { i18n } from '@kbn/i18n';
+import { DataSetup } from '../../../../../../../src/legacy/core_plugins/data/public';
```

Comment:
I wonder if we should have a single types file in our project that imports from wonky paths like this, and then we can simply reference that, rather than have lots of external references in various files in our project?

Reply:
I think this is excessively confusing because this is only importing the type definition. I'll see what I can do but don't expect to make a big change in this PR. There's some active discussion of relative imports like this in #40446